exporter/awss3exporter/README.md

```diff
@@ -25,6 +25,7 @@ The following exporter configuration parameters are supported.
 | `s3_partition` | time granularity of S3 key: hour or minute            | "minute" |
 | `file_prefix`  | file prefix defined by user                           |          |
 | `marshaler`    | marshaler used to produce output data otlp_json       |          |
+| `aws_endpoint` | overrides the default AWS endpoint_url                |          |
```

Comment:
Why is it called `aws_endpoint`? Could it be just `endpoint`?

Reply:
It could. Maybe `endpoint_url` would be clearer? 
Could you please clarify what you mean by `constructing it from a region and an S3 bucket`.
ASAIK, The URL is passed to the session as a fixed string if exist.